dashboard/app/helpers/levels_helper.rb

```diff
@@ -91,7 +91,7 @@ def select_and_remember_callouts(always_show = false)
   # Options hash for all level types
   def app_options
     # Provide the channel for templated and applab levels.
-    set_channel if @level.try(:project_template_level) || @level.game == Game::APPLAB
+    set_channel if @level.try(:project_template_level) || @level.game == Game.applab
```

Comment:
Yes, fixing an existing bug.  Good call, I'll add a test.

I'm not sure why, but `game` exists on all levels whereas `project_template_level` does not:
```ruby
irb(main):001:0> Level.all.all?(&:game)
=> true

irb(main):002:0> Level.all.all?(&:project_level_template)
NoMethodError: undefined method `project_level_template' for #<Eval:0x007fbffd380508>
```